extensions/blocks/repeat-visitor/repeat-visitor.php

```diff
@@ -37,5 +39,5 @@ function jetpack_repeat_visitor_block_render( $attributes, $content ) {
 	}
 
 	// return an empty div so that view script increments the visit counter in the cookie.
-	return '<div class="wp-block-jetpack-repeat-visitor"></div>';
+	return '<div class="' . $classes . '"></div>';
```

Comment:
I think you need `esc_attr` here.

Reply:
Good call. Done in f8914baee41c2d38198a85abcbdd0dc4b4349cac